pkg/pipeline/pipeline.go

```diff
@@ -21,17 +21,18 @@ import (
 	"io"
 	"path/filepath"
 	"regexp"
-
-	"github.com/ksonnet/ksonnet/pkg/util/k8s"
-	"github.com/ksonnet/ksonnet/pkg/util/strings"
+	gostrings "strings"
```

Comment:
I'm going to say stdlib should win the contest of _which package doesn't get aliased_.

Reply:
Why? I don't think there is a hard and fast rule.

Reply:
General familiarity and principal of least surprise. But this is a matter of preference, and only a suggestion :)